docs/user/wireguard.md

```diff
@@ -1,31 +1,59 @@
 # Remote access
 
+When you visit a website (or the Brewblox UI), your browser sends multiple requests to the server.
+This includes requests for current block values and graph data, but also requests for block changes.
+For this to be safe, the server must be able to trust the client.
+In a local network, this is simple: everyone on the network is trusted.
+
+This is no longer true when connected to the internet.
+Here, every single request must include some form of authentication to confirm that the sender can be trusted.
+
 Secure password-protected remote access is something we'd like to implement sometime in the future, but we're not there yet.
 In the meantime, a safe alternative is to install and use a Virtual Private Network (VPN).
 
-This tutorial uses [Wireguard](https://www.wireguard.com/) because it's fast, and easy to set up on a Raspberry Pi.
+This tutorial uses [Wireguard](https://www.wireguard.com/) because it's fast, and easy to set up.
 
 ::: tip
+This guide was written for Raspberry Pi OS or Ubuntu.
+
 Synology users can follow [this guide](https://github.com/runfalk/synology-wireguard) for server installation.
 :::
 
 ## What is a VPN?
 
-When you visit a website (or the Brewblox UI), your browser sends multiple requests to the server.
-This includes requests for current block values and graph data, but also requests for block changes.
-For this to be safe, every single request must include some form of authentication to confirm it's really you sending the request.
-
 A VPN creates a secure tunnel between your device and the server.
```

Comment:
"..between our device and the Pi. All data to the Brewblox UI goes through the tunnel, and is safe from reading and manipulation by other users on the internet. This tunnel can both be used for the Brewblox UI, and the SSH terminal.